content/altinn-studio/concepts/payment/_index.nb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Betaling
+description: Det er mulig å konfigurere brukerbetaling for tjenester som er gebyrbelagte eller der det er tillatt å kreve betaling.
+weight: 10
+---
+
+Betaling er et prosessteg som kan legges til applikasjonen. 
+Ved inngang til betalingssteget omdirigeres brukeren til en ekstern betalingsleverandør, og returneres til tjenesten når betalingen er fullført.
+
+Ordregrunnlag og detaljer om betalingen lagres i et dataelement som JSON (paymentInformation.json), og kan lastes ned via API etter insending av skjemaet. Det blir også generert en PDF-kvittering.
+Det er tjenesteeier som må håndtere oppbevaring av betalingsinformasjon i henhold til bokføringsloven.
+
+<object data="payment.drawio.svg" type="image/svg+xml" style="width: 100%;"></object>
+
+## Nets Easy
+
+Støtte for å bruke Nets Easy som betalingsleverandør følger med applikasjonen. Guide for oppsett av Nets Easy finner du [her](/nb/altinn-studio/guides/payment/).
```

Comment:
Det følger med støtte for å bruke Nets Easy som betalingsleverandør, med applikasjonen. Du finner veiledning for å sette opp Nets Easy finner du [her](/nb/altinn-studio/guides/payment/).

Tilleggskommentar: Her bør du ha lenke på hele setningen "Du finner...", ikke bare på "her".